### PR TITLE
BrenKT-SK815-TRANS-WEB-Email-alloffers-reset_account

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image 3948.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3949.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3936.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Empty alt attribute for image 3953.png in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/all_about_username/guide.nl-nl.md
WARNING: Other images have empty alt attributes
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/emails/email_reset_account/guide.en-gb.md
  | ScannerError: while scanning for the next token
found character '`' that cannot start any token
  in "<unicode string>", line 1, column 8:
    title: `Delete an email account`
           ^
CRITICAL: RuntimeError: Error encountered
```

**Review by bs-ovh — APPROVED** [review]

File: pages/web/emails/email_reset_account/guide.es-es.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- Eliminar una dirección de correo electrónico que ya no esté utilizando. 
+- Restaurar una cuenta de correo para utilizarla en una nueva dirección de correo. 
+- Restaurar una cuenta de correo para darlo de baja.
+
+**Cómo eliminar o restaurar una dirección de correo en un servicio de correo**
+
+## Requisitos
+
+- Disponer de una solución de correo electrónico de OVHcloud previamente configurada (**MX Plan**, incluida en nuestros [planes de hosting](https://www.ovhcloud.com/es-es/web-hosting/), incluida en un [alojamiento Start10M gratuito](https://www.ovhcloud.com/es-es/domains/free-web-hosting/) o contratada por separado como solución autónoma, como [**Hosted Exchange**](https://www.ovhcloud.com/es-es/emails/hosted-exchange/) o [**Email Pro**](https://www.ovhcloud.com/es-es/emails/email-pro/)).
+- Estar conectado al [área de cliente de OVHcloud](	https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}, en la sección `Web Cloud`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_reset_account/guide.es-es.md]:
```suggestion
- Estar conectado al [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}, en la sección `Web Cloud`{.action}.
```

File: pages/web/emails/email_reset_account/guide.pl-pl.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+### Usuń lub zresetuj konto e-mail
+
+Wybierz kartę odpowiadającą Twojej usłudze e-mail:
+
+> [!tabs]
+> **E-maile MXplan**
+>>
+>> Aby sprawdzić, czy Twoja oferta MXplan jest wersją historyczną czy nową, zapoznaj się z tabelą w części "[Sprawdź ofertę MXplan](#whichmxplan)" tego przewodnika.<br><br>
+>>
+>> Kliknij `E-maile`{.action}, po czym wybierz odpowiednią usługę MX Plan. Przejdź do zakładki `Konta e-mail`{.action}. Pojawi się okno, w którym widoczne są istniejące konta e-mail. Kliknij przycisk po prawej stronie konta, które chcesz zmienić, a następnie kliknij <i class="icons-ellipsis icons-border-rounded icons-masterbrand-blue"></i>Usuń konto` `{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_reset_account/guide.pl-pl.md]: action formatting needs fixes

File: pages/web/emails/email_reset_account/guide.pl-pl.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+>>![email](images/email-mxplan-legacy-reset.png){.thumbnail}<br>
+>>
+> **E-maile MXplan nowa wersja**
+>>
+>> Aby sprawdzić, czy Twoja oferta MXplan jest wersją historyczną czy nową, zapoznaj się z tabelą w części "[Sprawdź ofertę MXplan](#whichmxplan)" tego przewodnika.<br><br>
+>>
+>> Kliknij `E-maile`{.action}, po czym wybierz odpowiednią usługę MX Plan. Przejdź do zakładki `Konta e-mail`{.action}. Pojawi się okno, w którym widoczne są istniejące konta e-mail. Kliknij przycisk po prawej stronie konta, które chcesz zmienić, a następnie kliknij <i class="icons-ellipsis icons-border-rounded icons-masterbrand-blue"></i>Zresetuj to konto` `{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_reset_account/guide.pl-pl.md]: same error

File: pages/web/emails/email_reset_account/guide.pt-pt.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+>>
+>> Clique em `E-mails`{.action} e escolha o nome do serviço MX Plan em causa. Aceda ao separador `Contas de e-mail`{.action}. Na nova janela, podem ver-se as contas de e-mail existentes. Clique no botão <i class="icons-ellipsis icons-border-rounded icons-masterbrand-blue"></i> à direita da conta a modificar e, a seguir, em `Eliminar conta`{.action}.<br><br>
+>>![email](images/email-mxplan-legacy-reset.png){.thumbnail}<br>
+>>
+> **E-mails MXplan nova versão**
+>>
+>> Para verificar se a sua oferta MX plan é uma versão histórica ou nova, consulte a tabela da parte "[Identifique a sua oferta MXplan](#whichmxplan)" deste manual.<br><br>
+>>
+>> Clique em `E-mails`{.action} e escolha o nome do serviço MX Plan em causa. Aceda ao separador `Contas de e-mail`{.action}. Na nova janela, podem ver-se as contas de e-mail existentes. Clique no botão <i class="icons-ellipsis icons-border-rounded icons-masterbrand-blue"></i> à direita da conta a modificar e, a seguir, em `Reinicializar esta conta`{.action}.<br><br>
+>>![e-mail](images/email-mxplan-new-reset.png) {.thumbnail}<br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_reset_account/guide.pt-pt.md]:
```suggestion
>>![e-mail](images/email-mxplan-new-reset.png){.thumbnail}<br>
```

**Pull request merged**